docs/mgmt/mitigate_breaking_changes.md

```diff
@@ -0,0 +1,108 @@
+# Mitigate Breaking Changes
+
+## Overview
+
+Whenever an update is applied to an existing SDK library, there is a chance that the update will introduce breaking changes. Breaking changes are changes that are not backwards compatible and may require client code to be updated. For example, a method may be removed, a method may be renamed, or a method may have its signature changed. Breaking changes are inconvenient for clients that need to update their code in order to consume the latest version of the SDK. Therefore we would like to eliminate breaking changes whenever possible.
+
+To enforce this, we have a check in `dotnet build` that will fail if a breaking change is detected. This check is performed by the APIComPat tool. The tool is run as part of the build process and will compare the current assembly with the previous assembly (if it exists). If any breaking changes are detected, the build will fail, and you will see the following error message:
+```
+error : MembersMustExist : Member '<MemberName>' does not exist in the implementation but it does exist in the contract.
+```
+
+## General Principles
```

Comment:
Can refer https://azure.github.io/azure-sdk/general_introduction.html

Reply:
Do we have a section for breaking changes in this doc? I only find the SDK design guidelines which are not very relevant to the topic here

Reply:
Here https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes

Reply:
Since this is a public repo and this is an internal material, I do not like to include this in this document.
I am kind of focusing on "how do we resolve the build failures reported by ApiCompat tool in azure-sdk-for-net" in this document